flocker/node/functional/test_docker.py

```diff
@@ -138,6 +140,46 @@ def start_container(self, unit_name,
 
         return d
 
+    def test_force_remove(self):
+        """
+        ``DockerClient.remove`` can forcefully remove a running container
+        when the ``force`` parameter is set to True.
+        """
+        client = DockerClient()
+        unit_name = u'test-force-remove'
+        expected_states = (u'active',)
+        d = client.add(
+            unit_name=unit_name,
+            image_name=u'clusterhq/flask'
+        )
+        d.addCallback(
+            lambda _: wait_for_unit_state(
+                client, unit_name, expected_states
+            )
+        )
+
+        def check_is_listed(_):
+            listed = client.list()
+
+            def check_unit(results):
+                self.assertTrue(unit_name in [unit.name for unit in results])
```

Comment:
`assertIn(unit_name, [unit.name for unit in results])`

This provides a better failure message because it has both the "needle" and the "haystack".  `assertTrue` just has `True` or `False`.